src/_pytest/pytester.py

```diff
@@ -1278,7 +1278,7 @@ def spawn_pytest(
 
         """
         basetemp = self.tmpdir.mkdir("temp-pexpect")
-        invoke = " ".join(map(str, self._getpytestargs()))
+        invoke = " ".join([*map(str, self._getpytestargs()), *(args or [])])
```

Comment:
I'd have written this using `itertools.chain` instead of the `*`-syntax and I also tend to do explicit `is None` instead of the `or` trick.  But that's not to say there's anything wrong with this.

Reply:
```
if args is None:
    args = []

invoke = " ".join([*map(str, self._getpytestargs()), itertools.chain(args)])
```
Something like this?

Reply:
```
if args is None:
    args = []
" ".join(itertools.chain(map(str, self._getpytestargs()), args)
```
is what I had in mind, otherwise you still have the (to me) weird `[*map()]` thing.

But honestly, that's me and may not represent the current internal pytest style.  It all achieves the same thing really.

Reply:
Woops, missed the second unpacking there